### DISCUSSION TITLE
X (Twitter) visibility

### DISCUSSION THREAD

**Discussion by dmarro89** [dmarro89/dare-db#44] [Announcements]
Just to share with you some stuff I've found on X talking about dare-db.

https://x.com/godiscussions/status/1782046518251229449?s=46&t=VTCQwAnhMbK8aUQwI9tzxw

https://x.com/golangch/status/1837101176673718391?s=46&t=VTCQwAnhMbK8aUQwI9tzxw

Please feel free to comment and repost.
@vdmitriyev @TheInvincibleRalph

**Comment by vdmitriyev** [comment]
@dmarro89 thanks for the info. I have already seen and reposted it yesterday. 
First I was surprised to see a familiar dinosaur on the picture and later have realized, that I know, that is the tweet post is about 😃

**Comment by dmarro89** [comment]
I've also written a story on medium about the hashtable implementation -> https://medium.com/@davideraffaele.marro/redis-hash-tables-implementing-efficient-hash-tables-in-go-feff678dcb50

**Reply by vdmitriyev** [reply]
nice, that you for sharing!
